src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.Arrays;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private Truck truck = new Truck();
+    private Truck truck1 = new Truck();
+    private Truck truck2 = new Truck();
```

Comment:
Consider adding access modifiers to the `truck`, `truck1`, and `truck2` fields. Exposing fields directly is not a recommended practice. Instead, they should be private and accessed through getters and setters.